conf/examples/vault_external_standalone_mode.yaml

```diff
@@ -4,6 +4,8 @@ Example yaml for an external vault standalone OCS deployment
 ENV_DATA:
   encryption_at_rest: true
   vault_deploy_mode: external
+  # whether to use vault namespace or not
```

Comment:
Add some note that it's possible only for enterprise license

Reply:
done